src/doc/trpl/closures.md

```diff
@@ -1,9 +1,10 @@
 % Closures
 
-Rust not only has named functions, but anonymous functions as well. Anonymous
-functions that have an associated environment are called ‘closures’, because they
-close over an environment. Rust has a really great implementation of them, as
-we’ll see.
+Sometimes it is useful to wrap up a function and free variables for better
+clarity and reuse. The _free variables_ that can be used come from the
```

Comment:
I would put the hightlight on the first usage rather than the second, I think, but I can't quite say why